docs/development/releasing/index.md

```diff
@@ -5,13 +5,13 @@
 ### 1. Build and test locally:
 
 ```console
-$ DEMO=r-d-r-roofit EXCLUDE_COMPONENTS=r-ui,r-m-broker make ci
+$ reana-dev run-ci -c r-d-r-roofit --exclude-components=r-ui,r-m-broker
```

Comment:
[`reana-dev run-ci` will have required `--admin-email` and `--admin-password` options](https://github.com/reanahub/reana/issues/394) as well. Could you please update these examples?

Reply:
Done ✅ 

I also added them to the `cluster-deploy` command example, as they are mandatory over there too.